Chapter02/Activity2.03/app/src/main/java/com/example/loginactivity/MainActivity.kt

```diff
@@ -43,8 +61,8 @@ class MainActivity : AppCompatActivity() {
                     welcomeIntent.putExtra(PASSWORD_KEY, passWord)
 
                     //Reset text fields to blank
-                    user_name.text.clear()
-                    password.text.clear()
+                    this.userName.text.clear()
```

Comment:
@drulabs - Same here as for Activity 2.02 for changing the name of the local variable. Thanks